### PR TITLE
Increased gun ports on some human interceptors to encourage blaster use

### PR DESCRIPTION
**Pull request by Amazinite**
* Increase the number of gun ports that some interceptors have in order for them to take better advantage of the small size but considerable power density of blasters
  * The increase to interceptor gun ports is outlined in #3264. The Sparrow and Wasp gain the ability to install 2 more blasters, and the Fury gains the ability to install 2 more modified blasters OR 1 more beam laser/gatling gun with this PR.
  * New Wasp and Sparrow graphics by @KiLEdEnNis. New Fury graphic by @Kryes-Omega. 2x images at the ready and assets linked below.
  * No additional variants have yet been added that would take advantage of these increased gun ports and show the player how one could manage to say fit 4 blasters on a Sparrow and still run smoothly.
  * I opted to exclude the Berserker change as the description implies that one of the reasons it is falling out of popularity is due to its lack of versatility, which a lack of gun ports in order to saturate its weapon space with blasters would support.
  * The Hawk has also received no changes due to the fact that it currently seems like an interceptor designed to carry anything BUT blasters, seeing as how it has the same weapon space as the Fury but half the gun ports.

### CODE REVIEW

**Comment by KiLEdEnNis** [comment]
I don't really like how the Fury has 2 gun ports on 2 of its visible hardpoints and 1 on the other 2, and I think the Interceptors should get more visible guns in their models instead of having them pasted on top of the existing ones

**Comment by Amazinite** [comment]
>I don't really like how the Fury has 2 gun ports on 2 of its visible hardpoints and 1 on the other 2, and I think the Interceptors should get more visible guns in their models instead of having them pasted on top of the existing ones

Sounds like a job for someone who knows how to use Blender.

**Comment by KiLEdEnNis** [comment]
I'll get to it tomorrow if no one else wants to, shouldn't take long

**Comment by KiLEdEnNis** [comment]
the Blends, GIMP files, @2x & default images are at https://drive.google.com/drive/u/0/folders/1KnMBPL3zknJ_S2g7ootV4tPR12w5Tuym

File: data/ships.txt [hunk 1]
```diff
@@ -1421,6 +1421,8 @@ ship "Fury"
 	engine 17 35
 	gun -10 -26
 	gun 10 -26
+	gun -10 -26
+	gun 10 -26
```
Comment (KiLEdEnNis) [inline data/ships.txt]: new ship image gun hardpoints: (-)36 7

File: data/ships.txt [hunk 2]
```diff
@@ -1679,6 +1681,8 @@ ship "Hawk"
 	engine 24 41
 	gun -9 -17
 	gun 9 -17
+	gun -9 -17
+	gun 9 -17
```
Comment (KiLEdEnNis) [inline data/ships.txt]: new ship image gun hardpoints: (-)15 -17

File: data/ships.txt [hunk 3]
```diff
@@ -2588,6 +2592,8 @@ ship "Sparrow"
 	engine 5 35
 	gun -7 -10
 	gun 7 -10
+	gun -7 -10
+	gun 7 -10
```
Comment (KiLEdEnNis) [inline data/ships.txt]: new ship image gun hardpoints: (-)13 -10

File: data/ships.txt [hunk 4]
```diff
@@ -2868,6 +2874,8 @@ ship "Wasp"
 	gun 0 -38 "Javelin Pod"
 	gun -13 -31 "Energy Blaster"
 	gun 13 -31 "Energy Blaster"
+	gun -13 -31
+	gun 13 -31
```
Comment (KiLEdEnNis) [inline data/ships.txt]: new ship image gun hardpoints: (-)20 -26

**Comment by Amazinite** [comment]
The lighting is incorrect on the Fury that Kile gave me. He linked everything he used above, so if anyone wants to correct that real quick as Kile isn't online right now, that'd be appreciated.

**Comment by Kryes-Omega** [comment]
https://drive.google.com/file/d/1j-dGi2yVYkp9gD22doQqXjbeKvwA-Ipv/view?usp=sharing
Corrected fury assets

**Comment by Pointedstick** [comment]
Nice! How about the Finch and the Barb? Oughtn't they also get some more hardpoints--especially the Finch?

**Comment by KiLEdEnNis** [comment]
thanks Omega, I just copied the guns in the Blend so I blame MZ for not saving it correctly

**Comment by Amazinite** [comment]
>Nice! How about the Finch and the Barb? Oughtn't they also get some more hardpoints--especially the Finch?

On the Barb: the Barb is designed to carry a Proton Gun. It has a single gun port and does its job well. I don't see a need for increasing its gun port space (especially because it already has the highest weapon space among fighters).

On the Finch: if anything, I'd actually be for buffing the Lance and Dagger by giving them more gun ports. The Finch, Lance, and Dagger all share the same # of gun ports and same amount of weapon space right now, but the Finch is a clearly superior fighter with its 10 extra engine space and 10 to 20 extra outfit space. Perhaps the Lance and Dagger can each get 3 gun ports instead of 2.

**Comment by Pointedstick** [comment]
Or even the full four... Those fighters could easily mount four blasters if their heat and energy costs were more reasonable. Or three blasters and a Meteor Missile Launcher and four missiles with https://github.com/endless-sky/endless-sky/pull/3283!

**Comment by Amazinite** [comment]
>Or even the full four... Those fighters could easily mount four blasters if their heat and energy costs were more reasonable.

Just testing with the current numbers from this PR, 4 blasters on the Lance or Dagger is actually completely doable and fairly sustainable (as the fighters usually aren't firing 100% of the time)... you just have to remove the shield generator in order to fit a fuel cell big enough to power the blasters. The actual cost of the modifications to a default fighter are actually really cheap though (+21k credits on the Dagger and +18k on the Lance blaster builds I made), and I think the loss of a shield generator is perfectly reasonable for the increase in damage that 4 blasters provides over 2 laser beams (60 more shield DPS and only 36 less hull DPS).

**Comment by KiLEdEnNis** [comment]
just an idea; maybe add an even smaller, say, 5-ton shield generator for fighters? (of course less efficient than every other one)

**Comment by Nescio0** [comment]
Regarding the blaster improvements, no objections.
However, I fail to see why the number of gun slots has to be doubled for these interceptors. For comparison, the Behemoth has only two gun slots, as do quite a few light and medium warships; heavy warships such as the Bactrian, Derecho, Dreadnought, Falcon, and Leviathan have four. If all interceptors are allowed to have at least four gun ports to be able to choose between two lasers or four blaster, then shouldn't all other warships have significantly more gun ports as well, to choose between a few heavy or several lighter guns?

**Comment by Pointedstick** [comment]
I agree with you @Nescio0, but baby steps. :)

Also, from a certain perspective why fighter-sized craft would focus more on guns, while larger ships would focus more on turrets. Of course that implies that larger ships really should have more turret mounts...

**Comment by Amazinite** [comment]
>Regarding the blaster improvements, no objections.
However, I fail to see why the number of gun slots has to be doubled for these interceptors. For comparison, the Behemoth has only two gun slots, as do quite a few light and medium warships; heavy warships such as the Bactrian, Derecho, Dreadnought, Falcon, and Leviathan have four. If all interceptors are allowed to have at least four gun ports to be able to choose between two lasers or four blaster, then shouldn't all other warships have significantly more gun ports as well, to choose between a few heavy or several lighter guns?

Small ships are suppose to have small guns. The problem right now is that no ships have enough gun ports to fill up on Blasters. Once you get into larger warships though, they're intended to use larger weapons, which they already have enough gun ports to use. 

The best comparison I can make is to EVE online. I'm going off the top of my head right now, but in that game there are frigates, destroyers, cruisers, battlecruisers, and battleships (in order of increasing ship size). Imagine that destroyers in EVE are like interceptors in ES. In EVE, destroyers have space for up to 8 guns, but the next category up, cruisers, may only have space for 3 or 4. The reason for this is because destroyers and cruisers use different sized guns, destroyers using light weaponry and cruisers using medium weaponry. If a cruiser decided to fill up on light weaponry, it wouldn't even out damage a destroyer.

This is essentially the same scenario. We have smaller classed ships (the interceptors) and larger classed ships (light warships+). There are smaller classed weapons (blasters and beam lasers) and larger classed weapons (cannons). A light warships fit with larger weapons will out damage an interceptor full of blasters, but a light warship will not be able to fill up on blasters weapons and out damage an interceptor. (At least for the most part, because some light warships right now actually could fill up on more blasters than some interceptors.)

The goal of this PR is to make blaster guns and only blaster guns more usable. (This is why Pointed is claiming that this PR is a step in the right direction but isn't enough, because he wants blaster turrets to be more usable as well). Blaster guns are currently only viable weapons on interceptors that can't fit heavier weapons, but right now interceptors don't have enough gun ports to truly take advantage of the power density of blasters.

**Comment by Nescio0** [comment]
> Small ships are suppose to have small guns. [...] Blaster guns are currently only viable weapons on interceptors that can't fit heavier weapons

They can fit one or two heavier weapons. Let's make a comparison of the current weapon capacities and gun slots of interceptors:
```
Berserker | 35 | 4 (--)
Fury      | 40 | 4 (+2)
Hawk      | 40 | 2 (+2)
Sparrow   | 20 | 2 (+2)
Wasp      | 28 | 3 (+2)
```
Berserker, Fury, and Hawk can be equipped with the heaviest human guns and launchers. The Wasp can not equip a Particle cannon (size is 30) or a Typhoon Launcher (size is 35), but pretty much everything else. The Sparrow, which is considerably cheaper, is the only one which can not have heavy weapons.

(By the way, why is the Berserker the only interceptor which does not obtain two additional gun ports in your proposal? Its characteristic used to be that it had more than other interceptors.)

> right now interceptors don't have enough gun ports to truly take advantage of the power density of blasters.

Nor does any of the heavier warships, which are significantly more expensive. Increasing the number of gun ports unilaterally for interceptors only without looking at warships probably unbalances the game.

**Comment by gunqqer** [comment]
The idea of having pseudo tiers for weapons is a good one. You have small, weak weapons that are equipped on smaller ships en mass, and big, strong weapons that are equipped on bigger ships, but only a few.

**Comment by Pointedstick** [comment]
> Nor does any of the heavier warships, which are significantly more expensive. Increasing the number of gun ports unilaterally for interceptors only without looking at warships probably unbalances the game.

I would put it more than it gets us closer to _correctly_ balancing the game: fighter-sized craft armed solely with guns are pathetically bad right now unless deployed in huge swarms (50+) that are never actually seen in normal gameplay. The only way to make fighters in particular perform effectively is to cram them with missile launchers, which gives them an unrealistic missile capacity and makes them _too_ good--and also triggers really annoying missile-firing AI behaviors (https://github.com/endless-sky/endless-sky/issues/1537).

Allowing fighters to be armed with 4-6 small guns and 0-2 small low-capacity missile launchers firing large and effective missiles would alleviate both problems.

**Comment by Amazinite** [comment]
>They can fit one or two heavier weapons. Let's make a comparison of the current weapon capacities and gun slots of interceptors:

Some interceptors can fit a single heavier weapon. The only heavy weapon that some interceptors can fit two of is the Heavy Rocket. Even still, the ability to only fit a single heavier weapon (and 2 heavier weapons for only one type of heavy weapon on only two interceptors) on interceptors is by no means comparable to the fact that light warships can easily fit 2 to 4 of the larger human weapons, which they already have the gun ports to support. 

>(By the way, why is the Berserker the only interceptor which does not obtain two additional gun ports in your proposal? Its characteristic used to be that it had more than other interceptors.)

Mentioned in the PR description

>>right now interceptors don't have enough gun ports to truly take advantage of the power density of blasters.

>Nor does any of the heavier warships, which are significantly more expensive.

As mentioned, larger warships aren't suppose to be fitted with blaster weapons, and therefore they shouldn't gain an increase in gun ports for the purpose of fitting more blaster weapons. What warships do have is the ability to fill their hardpoints with heavier weapons, so they don't need an increase in hardpoint space to fit weapons they aren't designed to be using. That goes back to the EVE comparison; a cruiser full of small weapons isn't going to outdamage a destroyer with the same weapons because the cruiser is suppose to use medium weapons which allow the cruiser to easily out damage a destroyer. Translation to Endless Sky: a light warship full of blasters won't nor should it out damage an interceptor full of blasters (at least in general under this PR), but as soon as the light warship equips larger weapons, no single interceptor will stand a chance, even if the interceptor tries to equip heavier weapons due to the fact that the light warship is going to be able to equip more.

>Increasing the number of gun ports unilaterally for interceptors only without looking at warships probably unbalances the game.

Can you show how they would unbalance the game? As mentioned, this mainly only affects the number of blasters that can be equipped, and in some cases the number of beam lasers or meteor missiles. This isn't going to suddenly cause interceptors to outclass light warships.

**Comment by Amazinite** [comment]
I'd also like to mention the fact that an increase of 2 gun ports on some interceptors in order for them to fill their gun ports with blasters is MINIMAL compared to the **6 to 24** additional gun ports a **light warship** alone would need to achieve the same thing.

**Comment by Nescio0** [comment]
Likewise, to fully equip the Hawk with blasters you need an increase from two gun ports to eight.
To clarify, I'm not opposed to making blasters more desirable, and I fully support the proposed energy and heat improvements. However, I doubt adding gun ports to interceptors will achieve the desired effect. I believe it's not necessary for this patch and its merits are questionable.

Actually I am not even opposed to adding more gunports to interceptors per se; I'm opposed to overhauling interceptors without looking at warships.

**Comment by Amazinite** [comment]
>Likewise, to fully equip the Hawk with blasters you need an increase from two gun ports to eight.

Yeah, that was outlined in #3264; it's a matter of not giving interceptors TOO many gun ports (wanted to max at 6), and I only gave enough to the Fury to cap on modified blasters and enough to the Hawk to cap on meteor missiles.

>Actually I am not even opposed to adding more gunports to interceptors per se; I'm opposed to overhauling interceptors without looking at warships.

What would you suggest doing to the warships then? I was planning on analyzing the weapon space vs port space on the warships later, but just glancing at it so far, I'm not sure what there is to do.

**Comment by jafdy** [comment]
@Nescio0 What I am seeing in your arguments is a lack of actual arguments. While a simple "I don't think this will work" statement is sometimes perfectly reasonable, most of the time you need to think about the consequences of the change and highlight important points and coming up with counter proposals. You also need to consider the things everyone else is saying, and not just ignoring them because it doesn't fit your argument.

**Comment by Nescio0** [comment]
The reason for the increase in interceptor gun ports, as outlined in the first post, is to allow them to equip more blasters. However, to saturize them with blasters even more gun ports are required in some cases. Furthermore, those additional gun ports might as well be used for lasers, gatling guns, or other weapons, which means that this specific change does not necessarily result into blasters being used more often.
In short, I agree with the objective (improve usability of blasters), I don't disagree with the change per se (more gun ports), I just question and doubt this specific change will achieve the stated objective.

What I would suggest is to do the blaster tweaks (which I consider an improvement) in this patch and postpone the gun port increases (which I consider an overhaul) to a different patch which also includes changes to warships. E.g. the Gunboat has a weapon capacity of 100 but only 2 gun ports and 1 turret. If interceptors can choose between two heavy lasers or many blasters, shouldn't warships be able to choose between a few cannons or several lasers? Or between turrets versus more guns? Some long term vision would be nice as well. E.g., do we want ES to follow EVE?

**Comment by MCOfficer** [comment]
you did read #3264 , did you?

> E.g. the Gunboat has a weapon capacity of 100 but only 2 gun ports and 1 turret.

The Gunboat is supposedly more of a scout/general purpose ship than an actual warship, however we don't have a category for that. It gets mostly used for scanning and at least once referred to as "scout class".

> ...and postpone the gun port increase [...] to a different patch which also includes changes to warships. [...] If interceptors can choose between two heavy lasers or many blasters, shouldn't warships be able to choose between a few cannons or several lasers? Or between turrets versus more guns?

So... do it, but do it later? The situation is the following: the fighter change is ready, the warship change is still theoretical. How is this an argument to not change the fighters now? Yes, warships might need a similiar buff - but that's not an issue of this PR.

> do we want ES to follow EVE?

in this instance, apparently. in general, probably not.

**Comment by Amazinite** [comment]
>The reason for the increase in interceptor gun ports, as outlined in the first post, is to allow them to equip more blasters. However, to saturize them with blasters even more gun ports are required in some cases. Furthermore, those additional gun ports might as well be used for lasers, gatling guns

Please read up on #3264. I mention how the only interceptors that see an increased ability to install anything other than blasters are the Hawk and Fury (as well as outlining the specific changes per interceptor). The Fury is only able to equip one more beam laser or gatling gun than currently, and I'm considering reverting the Hawk change anyways since its current intent seems to be to equip medium-to-heavy category weapons (Heavy Lasers and Heavy Rockets given the variants in game).

> do we want ES to follow EVE?

No one ever said it should, I've only been using EVE as an example of how blasters can be done right and are being done wrong currently.

**Comment by jafdy** [comment]
In game design there is thing wrong with taking some ideas from one game, (in this case we have fighters with light weapons and warships with heavier weapons). Most of our ques come from EV, but the goal is to create a good game and not a carbon copy of an existing one. This does not preclude using the ideas from other games. 

For me, I have my doubts if this change will make a big shift to more blaster, but some people will try it and I personally don't have a problem making interceptors more adaptable and by extension more attractive. The blaster changes are nice to see, I'm going to trust Amazinite's spreadsheets as I'm not good with balancing.

**Comment by Nescio0** [comment]
> you did read #3264 , did you?

> Please read up on #3264.

Yes, I did; the first post points out that ships in general could use more gun ports and turret mounts to take advantage of smaller weapons, with which I agree in principle. Amazinite's first reply there is a proposal to increase the number of gun ports for seven interceptors (of which four are included in this patch). 

> This does not preclude using the ideas from other games.

No objections. If the idea is decent and suitable for this game, then the source doesn't matter.

> For me, I have my doubts if this change will make a big shift to more blaster

Same here; the changes do not hurt, though.

> I personally don't have a problem making interceptors more adaptable and by extension more attractive.

Nor do I have objections to this. My point is merely that the following reasoning is flawed:
> The goal of this PR is to make blasters more usable
**therefore** 
it's necessary to increase the number of gun ports for these four interceptors as proposed


> The blaster changes are nice to see

Yes, I agree; as I stated earlier, I am in favour of this proposal.

**Comment by KiLEdEnNis** [comment]
@Nescio0 
short summary: more weapon ports should ideally give the player a choice between a few big or a lot of small guns, making a lot more weapons viable and thus increasing the options for players as they won't be limited to big guns for the most DPS

**Comment by MCOfficer** [comment]
so we're basically all agree but arguing anyways? just wanted to make sure i got that right 😄

**Comment by Amazinite** [comment]
I've reverted my changes to the Hawk.

Reasoning: The Hawk can currently be seen in game with two heavy lasers or two heavy rockets. Given the fact that it uses two weapons with "heavy" in the name, I'd dare to call it a heavy interceptor, designed to normally carry larger weapons than other interceptors. In order to give the Hawk more of an advantage over its interceptor siblings that gain more gun ports with this PR, I'd suggest increasing the Hawk's weapon space by 10, which allows it to install two torpedo launchers. I feel that that is out of the scope of this PR though, so I won't do it here.

**Comment by MCOfficer** [comment]
two torpedos pack quite a punch. with decent hit-and-run a hawk like this could even take out some light/medium warships 🤔

**Comment by Lineth** [comment]
@MCOfficer Is that entirely wrong? So long as it loses to other interceptors or fighters, I think making the Hawk a sort of informal 'bomber' is a good way of making it stand out for itself a bit.

**Comment by KiLEdEnNis** [comment]
making all interceptors the same makes them monotome, I like the idea of the Hawk being a "heavy Interceptor", maybe it should be a bit slower but tankier then but I think that would change it too much

**Comment by Pointedstick** [comment]
I like the idea of the Hawk being a dedicated fighter-bomber or torpedo boat type of craft. The small number of gun ports but large weapon space really makes that work.

**Comment by solardawning** [comment]
Re, hawk: Keep in mind that 50 weapon space would also mean dual plasma cannons, not just torpedoes.
The Hawk is already just a worse Arrow, though.
(Arrow: Faster, only requires one crew, has two gun and one turret port, and already has 50 weapon tonnage.)
The Arrow is the best interceptor. 🥇

**Comment by KiLEdEnNis** [comment]
@solardawning the Arrow hull alone (1,2M) costs more than stock Interceptors though, and a stock Arrow costs 3x as much, so it's actually quite a bit worse in cost-effectiveness

**Comment by Pointedstick** [comment]
I'm totally fine with heavy fighter type ships being able to mount dual plasma cannons.

**Comment by kikotheexile** [comment]
noooooooo

Their damage is too high, their heat output is too much, and their energy and cooling requirements are too much. A clipper is the smallest ship that can should be able to dual plasmas. Light warships fit the niche you are aiming for *perfectly* and having fighters mount them would further marginalize them to the point of even less people ever bothering to buy one.

Let variety be the spice of life, yo, don't further remove any semblance of balance by letting small ships use exactly the same weapons as heavy warships, arguably more effectively, at a fraction of the cost. Spam is already overpowered.

**Comment by MCOfficer** [comment]
i did some testing with a 50-outfit space hawk, and while you can fit it as plasma bomber, it has typical bomber-drawbacks. I tested with human outfits only, while constantly accelerating (no turning) & firing, Even with downgraded shields, my best loadout couldn't sustain more than 11 seconds due to overheating. The only decent loadout i found was with downgraded shields & engines, focusing entirely on energy regen & cooling, which slowed it down to almost stock-firebird speeds. You can build a plasma hawk like that, but it seems pretty balanced to me.

**Comment by Pointedstick** [comment]
Thinking about this some more from a flavor/player motivation perspective, the blaster already has the following (theoretical) advantages over lasers:
- Longer range
- Ability to fit more into the same tonnage of weapon space (if there are enough hardpoints available)
- Shield-heavy DPS profile

In practice the range isn't higher enough to matter; ships don't have enough gun and turret hardpoints to take advantage of the Blaster's favorable DPS/ton, and the Shield DPS itself isn't high enough.

This PR improves #2 for fighter-sized craft at least by adding more gun hardpoints, but by reducing the heat and energy costs, it may go too far into the territory of reducing the distinctiveness of the weapons, since efficiency is _supposed_ to be lasers' strong suit in the game lore. You should choose blasters if you don't care about efficiency and just want raw power! Right now you also reach for a laser to get that, which is what seems wrong to all of us.

How about instead further enhancing the blaster's existing strengths? You already do that in this PR by adding more gun slots, which is perfect. But rather than reducing the energy and heat costs, we might consider increasing the Blaster's range and shield DPS at the expense of armor DPS.

I'm thinking maybe 450 range (480 for the Mod Blaster) and 60 shield DPS/20 armor DPS (80/26 for the Mod Blaster).

With that change, the laser would be an anti-shield beast, but to deal with armor you might actually want some laser or missile weapons to finish them off, just like you do with the Sunbeam right now. Lopsided weapons are _fun_! They encourage tactical choices.

**Comment by Amazinite** [comment]
@kikotheexile I wouldn't *just* up the Hawk's weapon space, as that does blur the line between the Hawk and light warships. I'd likely reduce the heat dissipation of the Hawk or something similar so that using Plasma Cannons isn't a completely viable option on the Hawk, but using double torpedoes or double heavy lasers would still be viable. Although, the main reason I *wouldn't* up the Hawk's weapon space is because it could end up gaining double electron beams, but without testing that I'm not sure how much of a problem that would be.

**Comment by kikotheexile** [comment]
The hawk is already basically a light warship, or by comparison the quicksilver is basically an interceptor. Dual plasmas or ebeams is a bit much. That said, i am not a balance lord. The concept bothers me. Doing this would also mean a slight need for powercreep on the gunship and quicksilver and by extension the manta

**Comment by kikotheexile** [comment]
With the changes, that is*

But once again your call. I respect your decision

**Comment by endless-sky** [comment]
I'm still not convinced about adding hardpoints, but I've reduced the energy and heat cost of blasters in beb7f03.

**Comment by Amazinite** [comment]
Alright. This PR is only the hardpoint change now, so you can do with it as you wish.

**Comment by Pointedstick** [comment]
With the new Blaster changes, they're actually a decent enough choice that you might _want_ to stuff 4 or 6 in a fighter-sized ship!

**Comment by KiLEdEnNis** [comment]
more hardpoints generally mean a few strong or a lot of weaker weapons, but I'm fine w/ leaving this PR on hold (at least until all those weapon rebalances are done) to ensure we only open new playstyles, not forcing players into another clearly better than the rest strategy

**Comment by JannHL** [comment]
Currently only missileboats are a worthwhile build for small ships, but if the gunports get increased and @3372 gets added this would make blasters a good thing to put on em.

**Comment by Pointedstick** [comment]
It's a real shame this was never merged IMHO. The logic behind it is even more true now that blasters are better, but re-visiting this will require additional art asset changes to make it merge-able once again.

I still support this, and would also support doing the same for the Lance and Dagger too.

**Comment by Amazinite** [comment]
This PR is still linked as a card in the weapon balance project, so it isn't out of the picture yet. I've resolved the current conflict cause by the new leak mechanic, and it shouldn't be hard for me to get a hold of someone to make the shipyard thumbnails that match the increased guns.

**Comment by Amazinite** [comment]
I'm personally leaning away from this change now, especially after playing extensively with the missile changes. There's really no reason to *want* that many blasters.

**Comment by DingusShingleton** [comment]
> There's really no reason to _want_ that many blasters.

Fun?

**Comment by Pointedstick** [comment]
Another use for this PR would be to better enable https://github.com/endless-sky/endless-sky/issues/3372.

**Comment by Zitchas** [comment]
Could we re-open this, or would it be better to just recreate something along the same lines but new?

**Comment by Pointedstick** [comment]
I'd be in favor of that, but we'll need additional shipyard graphics, as those were added to the game after the PR was submitted.

**Comment by Pointedstick** [comment]
Either way, if @Amazinite isn't interested in the idea anymore, it might be better for someone else to take over.

**Comment by Zitchas** [comment]
I'd be happy to take over. Would you suggest just pulling the changes from here, or going a new direction?

**Comment by Pointedstick** [comment]
No preference. If you're doing the work, it's your call. :)

**Comment by Zitchas** [comment]
Later on I'll start a new PR with some changes based on these, but my own stuff. Unless Derpy opens this before then.

**Comment by Amazinite** [comment]
The models from this PR are outdated anyway, so I don't see a use in reopening this one just to then completely redo it. Might as well just open a new PR. Any new PR might also want to look at increasing the number of ports on human fighters/drones as well.

**Comment by Zitchas** [comment]
Sounds good. I'll work on opening a new PR, and then reference back to this one in case of relevant discussion and previous examples. And yes, definitely look at doing all the fighters and drones, not just the trio done here.

**Comment by Pointedstick** [comment]
Sounds like a good plan to me!

**Pull request closed without merging**